src/PowerSimulations.jl

```diff
@@ -146,8 +146,13 @@ export get_problem_results
 export get_system
 export get_system!
 export set_system!
+export list_dual_keys
```

Comment:
I added exports of `list.*keys` because of their potential use in PowerGraphics. But, I'm not convinced that they are required.

Reply:
These would be useful also for exploration, we should keep them in the meantime.